s4sim/hardware/config_1shooter.py

```diff
@@ -297,13 +297,13 @@ def get_example():
     bnd["A"] = 0.09
     bnd["C"] = 0.87
     bands["ULFL1"] = bnd
-
+    
     bnd = OrderedDict()
     bnd["center"] = 27.0
     bnd["low"] = 24.0
     bnd["high"] = 30.0
     bnd["bandpass"] = ""
-    bnd["NET"] = 177.0
+    bnd["NET"] = 387.0
```

Comment:
And that. We'll certainly want to redo the noise simulation for the baseline simulation tool